packages/commercetools/theme/tests/e2e/integration/e2e-checkout-payment-validation.spec.ts

```diff
@@ -0,0 +1,26 @@
+import requests, { CreateCartResponse } from '../api/requests';
+import page from '../pages/factory';
+
+context(['regression'], 'Checkout - Payment', () => {
+  beforeEach(function () {
+    cy.fixture('test-data/e2e-checkout-payment-validation').then((fixture) => {
+      this.fixtures = {
+        data: fixture
+      };
+    });
+  });
+  it('Should successfully redirect to home page - shipping and billing address are not set', function () {
+    const data = this.fixtures.data[this.test.title];
+    page.checkout.payment.visit();
+    expect(page.home.path).to.be.eq(data.homePath);
```

Comment:
```suggestion
    cy.url().should('equal', Cypress.config.baseUrl)
```
Should be something like that. Cuz as it is, we don't compare the actual URL, but hardcoded values.

Reply:
Also, test data for this case can be removed after